ex03.ex

```diff
@@ -77,7 +80,9 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains(list, a) do
```

Comment:
### -4

The idea was to use pattern matching and no library functions (yes, `in` is a function in module Kernel).

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~

Going forward, `in` is obviously the right way to do this, but that's not the objective of this assignment.